tests/unit/model/designation-test.js

```diff
@@ -54,3 +54,191 @@ test('Relationships with other models', function(assert){
   assert.equal(relationshipOrdersPackage.key, 'ordersPackages');
   assert.equal(relationshipOrdersPackage.kind, 'hasMany');
 });
+
+test('check isLocalOrder', function(assert){
+  const model = this.subject({detailType: 'LocalOrder'});
+  assert.equal(model.get('isLocalOrder'), true);
+});
+
+test('check dispatchedItems returns items with sentOn not null', function(assert){
+  assert.expect(3);
+  const model = this.subject();
+  var store = this.store();
+  var item1 = null;
```

Comment:
@viveksriram1 , I don't think we need to initialise this as null, we can directly define them as var item1, item2 and it will take default value i.e undefined.
What do you think?